DFReader.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
So this doesn't *reeally* accomplish much, as DFReader is usually called from `mavlogdump.py` via `mavutil.py`.

We should probably change `mavlogdump.py`'s shebang line as part of this PR.